gandi/cli/commands/dnssec.py

```diff
@@ -0,0 +1,50 @@
+""" Domain namespace commands. """
+
+import click
+
+from gandi.cli.core.cli import cli
+from gandi.cli.core.utils import output_contact_info, output_domain, display_rows
+from gandi.cli.core.params import pass_gandi
+
+
+@cli.group(name='dnssec')
+@pass_gandi
+def dnssec(gandi):
+    """Commands related to dnssec."""
+
+
+@dnssec.command()
+@click.option('--flags', default=None, help='Flags')
```

Comment:
I think you need to specify the type here otherwise you'll get a string and the API will refuse it

Reply:
@romuald done